02-starting-with-data.Rmd

```diff
@@ -255,7 +253,7 @@ feature to get the full and correct column name.
 > ## 3.
 > surveys_middle <- surveys[nrow(surveys)/2, ]
 > ## 4.
-> surveys_head <- surveys[-c(7:nrow(surveys)),]
+> surveys_head <- surveys[-(7:nrow(surveys)), ]
```

Comment:
What is the notation that gets added? The use of ":" to create vectors of numeric values by increment of 1 already appears in the lesson (e.g. in "surveys[1:3, 7]"). 

I think that the use of `c` here is improper (even if it works) since `7:nrow(surveys)` is already a vector and the function of `c` is to form a vector from elements. So it is redundant and unnecessary (and not a really good practice).

Would you find `surveys_head <- surveys[-7:-nrow(surveys), ]` an easier form for the students?

Reply:
I like your additions on this section in your last commit: they split the novel concepts :) But I saw that you are keeping the c(xx:xx) :smile: Maybe I am wrong and it is an acceptable notation... :slightly_smiling_face: